bundle-workflow/README.md

```diff
@@ -216,3 +217,17 @@ The following options are available.
 |--------------------|-------------------------------------------------------------------------|
 | --keep             | Do not delete the temporary working directory on both success or error. |
 | -v, --verbose      | Show more verbose output.                                               |
+
+### Onboarding Requirements
+
+With many components included in the distribution each component has the responsibility to keep the release process streamlined and react to integration issues.
+
+- Components repositories must have a tag matching all manifest versions so issue can be appropriately tagged.
+- Component maintainers must be able to respond to blocking issue within 2 business days.
+- All incoming branches need continuous integration checks enabled and passing.
+  - If a component has defect that blocks the distribution, issues will be opened on that repository and depending on the circumstance the component might be rolled back
```

Comment:
This is not a requirement, it's a statement of what happens when things break, remove.